uber/custom_tags.py

```diff
@@ -105,6 +105,14 @@ def join_and(xs):
         return ', '.join(xs)
 
 
```

Comment:
On the main prereg form, we want to show the overall price of a thing, e.g.
- Add a Tshirt: $70
- Supporter: $100

However, on the other pages such as when claiming an already-paid-for group badge or filling out the badge transfer form, we want to just list the price added since otherwise people might think they're being charged the base badge price again:
- Add a Tshirt: +$20
- Supporter: +$50

This function looks at which page we're on and chooses between the two.